packages/common/src/controllers/AccountController.ts

```diff
@@ -334,7 +334,7 @@ export default class AccountController {
 
     if (response.errors.length > 0) throw new Error(response.errors[0]);
 
-    await this.reloadSubscriptions({ delay: 2000 });
+    await this.reloadSubscriptions({ retry: 5 });
```

Comment:
I try to understand the retry mechanism without a delay. When the `delay` is not provided, i do not see it defaults to a certain delay (which I assumed). So it retries directly after the network requests fails?

Reply:
When retry is set but no delay, a default delay of 1000s is used. I didn't want to use a default value in the argument, because that would also delay the first try.